app/src/main/java/com/example/tasky/agenda/data/remote/uri/PhotoExtensionParserImpl.kt

```diff
@@ -0,0 +1,24 @@
+package com.example.tasky.agenda.data.remote.uri
```

Comment:
You are parsing the file extension of a photo
That really doesn't have much to do with the "remote" package
So maybe have a photo_util package or similar inside data, where you store this class and PhotoByteConverterImpl?